fake_dependencies/capability-service/capability-data.json

```diff
@@ -79,5 +79,20 @@
     ],
     "contexts": [],
     "RootId": ""
+  },
+  {
+    "id": "0d03e3ad-2118-46b7-970e-0ca87b59a202",
+    "name": "Team One",
+    "description": "The team that came first",
+    "members": [
+      {
+        "email": "highlander@me.com"
+      },
+      {
+        "email": "jdog@me.com"
+      }
+    ],
+    "contexts": [],
+    "RootId": ""

```

Comment:
should RootId, be rootId?

Reply:
Seems to be RootId throughout the entirety of capability-data.json. RootId is nonetheless on the way out(AFAIK, feel free to correct me).